pom.xml

```diff
@@ -535,9 +535,20 @@
             <configuration>
               <excludes>
                 <exclude>**/CudaFatalTest.java</exclude>
+                <exclude>**/ColumnViewNonEmptyNullsTest.java</exclude>
               </excludes>
             </configuration>
           </execution>
+          <execution>
+            <id>non-empty-null-test</id>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <argLine>-da:ai.rapids.cudf.AssertEmptyNulls</argLine>
+              <test>**/ColumnViewNonEmptyNullsTest.java</test>
```

Comment:
I think build.sh also needs to be updated to handle this test that shouldn't run with other tests.  That script is explicitly pulling out CuFileTest and CudaFatalTest, probably needs to pull out this one as well.